gulpfile.js

```diff
@@ -33,6 +34,13 @@ const NODE_MODULES_PATH = ('NODE_PATH' in process.env) ?
     process.env.NODE_PATH.split(':')[0] :
     './node_modules/';
 
+const config = tryRequire('./debug-config.json') || {'sass-lint': false,
```

Comment:
nit: this line could use a little more spreading out, maybe semi-mirror the `debug-config.json-dist` like so:
```
const config = tryRequire('./debug-config.json') || {
  "sass-lint": false,
  "js-lint": false
};